source/elements/oneDPL/source/common.rst

```diff
@@ -5,7 +5,7 @@
 Namespaces
 ----------
 
-oneDPL uses :code:`namespace oneapi::std` for the subset of the standard C++ library for kernels,
-and uses :code:`namespace oneapi::dpl` for other functionality including Parallel STL algorithms,
-oneDPL execution policies, and extensions.
+oneDPL uses :code:`namespace oneapi::dpl` for all its functionality including Parallel STL algorithms,
+oneDPL execution policies, etc. For the subset of the standard C++ library for kernels, the standard class
+and function names aer also aliased in :code:`namespace oneapi::dpl`.
```

Comment:
probably it would be better to say "oneDPL uses :code:`namespace oneapi::dpl` for all its functionality including Parallel STL algorithms, oneDPL execution policies and the subset of the standard C++ library for kernels: the standard class and function names

Reply:
aer -> are if the previous suggestion wouldn't be applied

Reply:
Thanks for finding the typo. I think having a separate sentence about the kernel subset is easier read (due to shorter sentences) and puts focus on the specifics of the subset.